[2024-05-06T04:04:36.757+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: main.write_to_clean_data_task manual__2024-05-06T04:03:59.294928+00:00 [queued]>
[2024-05-06T04:04:36.772+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: main.write_to_clean_data_task manual__2024-05-06T04:03:59.294928+00:00 [queued]>
[2024-05-06T04:04:36.772+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2024-05-06T04:04:36.794+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): write_to_clean_data_task> on 2024-05-06 04:03:59.294928+00:00
[2024-05-06T04:04:36.809+0000] {standard_task_runner.py:57} INFO - Started process 494 to run task
[2024-05-06T04:04:36.817+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'main', 'write_to_clean_data_task', 'manual__2024-05-06T04:03:59.294928+00:00', '--job-id', '214', '--raw', '--subdir', 'DAGS_FOLDER/main.py', '--cfg-path', '/tmp/tmp253a1ss7']
[2024-05-06T04:04:36.823+0000] {standard_task_runner.py:85} INFO - Job 214: Subtask write_to_clean_data_task
[2024-05-06T04:04:36.909+0000] {task_command.py:415} INFO - Running <TaskInstance: main.write_to_clean_data_task manual__2024-05-06T04:03:59.294928+00:00 [running]> on host 6a3971e82789
[2024-05-06T04:04:37.039+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='main' AIRFLOW_CTX_TASK_ID='write_to_clean_data_task' AIRFLOW_CTX_EXECUTION_DATE='2024-05-06T04:03:59.294928+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-06T04:03:59.294928+00:00'
[2024-05-06T04:04:37.178+0000] {warnings.py:109} WARNING - /opt/***/dags/main.py:72: UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
  df = pd.read_sql_query(query, conn)

[2024-05-06T04:04:57.220+0000] {python.py:194} INFO - Done. Returned value was: None
[2024-05-06T04:04:57.242+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=main, task_id=write_to_clean_data_task, execution_date=20240506T040359, start_date=20240506T040436, end_date=20240506T040457
[2024-05-06T04:04:57.355+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-05-06T04:04:57.393+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
